server/cshr/services/users.py

```diff
@@ -0,0 +1,38 @@
+from server.cshr.models.users import User
+
+from django.contrib.auth.hashers import check_password
+
+
+def get_user_by_id(id: str) -> User:
+    """Return user who have the same id"""
+    try:
+        return User.objects.get(id=int(id))
+    except User.DoesNotExist:
+        return None
+
+
+def get_user_by_email_for_login(email: str) -> User:
+    """Return user who have the same email"""
+    try:
+        return User.objects.get(email=email)
+    except User.DoesNotExist:
+        return None
+
+
+def success_login_user(email, password) -> User:
+    """Return user who have the same email and password"""
+
+    user = User.objects.get(email=email)
```

Comment:
use here to get the user by id and make a condition on it because if the user email is wrong this step will return an error.
```suggestion
    user = get_user_by_email(email=email)
```